src/elements/molecules/Accordion/Accordion.example.jsx

```diff
@@ -29,13 +29,13 @@ export default [{
       name: 'Default styling',
       component: (
         <Accordion>
-          <Accordion__section title="Accordion section one" tabindex="0">
+          <Accordion__section title="Accordion section one" id="accordion-1">
```

Comment:
Lets move id up to be set once on the top <Accordion> tag so devs only have to supply this once at the top of the markup.